### PR TITLE
Freezes v3.3.1 and makes v3.3.2 WIP

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -179,6 +189,7 @@ chart-operator).
 
 
 
+[v3.3.1]: https://github.com/giantswarm/k8scloudconfig/commits/master/v_3_3_1
```
Comment (xh3b4sd) [inline CHANGELOG.md]: `v3.3.2` is missing, which you just added.

Comment (rossf7) [reply CHANGELOG.md]: Good catch. Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,14 +9,24 @@ The latest version is considered WIP and it is a subject of change. All other
...
+### Removed
+
+
+## [v3.3.1]
 
 ### Changed
 - Changed some remaining images to be pulled from Giant Swarm's registry.
 - Updated Alpine sidecar for Ingress Controller to version 3.7.
 - Fixed mkfs.xfs for containerized kubelet.
 - Updated hyperkube to version 1.10.3.
```
Comment (puja108) [inline CHANGELOG.md]: I still think this should say `Kubernetes` or at least `Kubernetes (hyperkube)` as hyperkube is just the name of the container and the component is actually Kubernetes.

Comment (rossf7) [reply CHANGELOG.md]: Yes, that makes sense to me as k8s is the component. I went with `Kubernetes (hyperkube)`.

**Pull request merged**